### PR TITLE
PLT-5721 Show Group Messages and correct channel info

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
PR includes the channel info options for group messages

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-5721

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Has UI changes
- [x] Includes text changes and localization file updates

#### Device Information
This PR was tested on: IOS 10.2.1 iPhone 6s, Android 6.0.1 Samsung J5

#### Screenshots
IOS
img_0339

Android
screenshot_20170320-175744

### CODE REVIEW

File: app/components/channel_drawer_list/channel_drawer_list.js [hunk 1]
```diff
@@ -125,7 +126,15 @@ class ChannelDrawerList extends Component {
 
     handleClose = (channel) => {
         this.setState({showOptions: false});
-        this.props.actions.closeDMChannel(channel);
+
+        switch (channel.type) {
+        case Constants.DM_CHANNEL:
+            this.props.actions.closeDMChannel(channel);
```
Comment (hmhealey) [inline app/components/channel_drawer_list/channel_drawer_list.js]: Should we maybe make a single action for closing a DM/GM channel that handles figuring out which type it is? I noticed you're also doing this below

File: app/scenes/channel_info/channel_info_container.js [hunk 2]
```diff
@@ -3,15 +3,26 @@
...
+import {getChannelStats, deleteChannel} from 'mattermost-redux/actions/channels';
+import {
+    getCurrentChannel,
+    getCurrentChannelStats,
+    getChannelsByCategory,
+    canManageChannelMembers
+} from 'mattermost-redux/selectors/entities/channels';
 import {getUser} from 'mattermost-redux/selectors/entities/users';
 
+import navigationSceneConnect from '../navigationSceneConnect';
```
Comment (hmhealey) [inline app/scenes/channel_info/channel_info_container.js]: Could you change this to not be relative? I'd like to avoid relative imports that go backwards

**Comment by enahum** [comment]
@hmhealey ready to re-review dude

**Comment by enahum** [comment]
I have to fix a conflict here and then I'll merge

**Pull request merged**